src/orion/core/cli/evc.py

```diff
@@ -15,8 +15,14 @@
 def _add_auto_resolution_argument(parser):
     parser.add_argument(
         "--auto-resolution",
+        help="Deprecated")
```

Comment:
```suggestion
        help="Deprecated. Conflicts are now resolved automatically by default. See --manual-resolution to avoid auto-resolution.")
```